### PR TITLE
[3.0.1] V3 bug fixes

### PR DESCRIPTION
**Pull request by worlddlckgh**
- Enforce initial input / output token fetching
- Revert scoped-preflight by tailwind, so it doesn't affect sites with tailwind
- update tags

### CODE REVIEW

File: src/components/FormPairRow.tsx [hunk 1]
```diff
@@ -30,7 +25,7 @@ export interface IPairRow {
 interface IMultiTag {
   isVerified: boolean;
   isLST: boolean;
-  isUnknown: boolean;
+  // isUnknown: boolean;
```
Comment (chiew256) [inline src/components/FormPairRow.tsx:28]: Can just remove this?

File: src/components/FormPairRow.tsx [hunk 2]
```diff
@@ -107,16 +94,10 @@ const MultiTags: React.FC<IPairRow> = ({ item }) => {
         </div>
       ))}
 
-      {isLST && (
```
Comment (chiew256) [inline src/components/FormPairRow.tsx:110]: Why we remove lst tag?

**Review by chiew256 — APPROVED** [review]

**Comment by llegoelkelo** [comment]
💪

File: src/components/FormPairRow.tsx (hunk 1 above)
Comment (worlddlckgh) [reply src/components/FormPairRow.tsx:28]: we're still experimenting what's the more approachable and understandable for our users

File: src/components/FormPairRow.tsx (hunk 2 above)
Comment (worlddlckgh) [reply src/components/FormPairRow.tsx:110]: we should standardise a "verified" tag instead of having so many tags that user might be confused.

**Review by wkitsin — COMMENTED** [review]
im so sorry, im late to the review.

**Comment by llegoelkelo** [comment]
@worlddlckgh I see the PR has been merged. When would this be released?

**Comment by worlddlckgh** [comment]
> @worlddlckgh I see the PR has been merged. When would this be released?

it should already be deployed.
just load main-v3.js

**Pull request merged**